src/main/java/by/itechart/api/util/annotation/ApiPageable.java

```diff
@@ -0,0 +1,20 @@
+package by.itechart.api.util.annotation;
+
+import io.swagger.annotations.ApiImplicitParam;
+import io.swagger.annotations.ApiImplicitParams;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ElementType.FIELD, ElementType.ANNOTATION_TYPE, ElementType.TYPE, ElementType.METHOD})
+@Retention(RetentionPolicy.RUNTIME)
+@ApiImplicitParams({
+        @ApiImplicitParam(name = "page", dataType = "integer", paramType = "query", defaultValue = "0", value = "Results page you want to retrieve (0..N)"),
```

Comment:
Don't we count pages starting from 1? Why default value is 0?
Another point: in value, it would be better to give more clear parameter description, e.g. `Page number you want to retrieve (starting from 1)`